### PR TITLE
NMS-12750: Fix delta switched for Netflow5

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12750

### CODE REVIEW

File: features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java [hunk 1]
```diff
@@ -110,6 +110,9 @@ public Integer getEngineType() {
 
     @Override
     public Long getDeltaSwitched() {
+        if (getNetflowVersion().equals(NetflowVersion.V5)) {
+            return flowMessageProto.hasFirstSwitched() ? flowMessageProto.getFirstSwitched().getValue() : null;
```
Comment (fooker) [inline features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java:114]: Maybe we can always use this as an fallback for `deltaSwitchen` without making it protocol specific

Comment (cgorantla) [reply features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java:114]: Yeah, If you think that would be correct protocol wise, I can change that.

File: features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java [hunk 2]
```diff
@@ -110,7 +110,7 @@ public Integer getEngineType() {
 
     @Override
     public Long getDeltaSwitched() {
-        return flowMessageProto.hasDeltaSwitched() ? flowMessageProto.getDeltaSwitched().getValue() : null;
+        return flowMessageProto.hasDeltaSwitched() ? new Long(flowMessageProto.getDeltaSwitched().getValue()) : getFirstSwitched();
```
Comment (fooker) [inline features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java:113]: I don't think the extra boxing with `new Long` is necessary here.

Comment (cgorantla) [reply features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java:113]: `ternary` operator tries to auto unbox null and encounters NPE.  WIth boxing the first expression it doesn't unbox the second expression.

**Pull request merged**